### PR TITLE
models setuped

### PR DESCRIPTION
**Pull request by Shakar-coder**
##  model sutup
these models are created
- [x] categories
- [x] bikes
- [x] carts
- [x] orders
- [x] users
- [x] reviews

there was a mistake in "reviews" file name
 i wrot it like this (rviews) but it is fixed.

### CODE REVIEW

**Comment by fanfanlulu** [comment]
Let's double check if the propertities are correctly defined with teachers

**Review by fanfanlulu — CHANGES_REQUESTED** [review]
Let's double check the properties with teachers

File: app/models/bike.rb
Comment (fanfanlulu) [inline app/models/bike.rb:1]: Add:
has_many :bikes

File: app/models/category.rb
Comment (fanfanlulu) [inline app/models/category.rb:1]: Add:
has_many :bikes

**Comment by Shakar-coder** [comment]
This branch is now obsolete

**Pull request closed without merging**